### PR TITLE
Michelle Cali

### PR DESCRIPTION
**Pull request by michellecali**

### CODE REVIEW

File: lib/conditionals.md [hunk 1]
```diff
@@ -5,8 +5,10 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
 
 # What is a conditional?
 
-Replace me with your response...
+A conditional is a statement which sets the parameters for which parts of the code will be executed.
```
Comment (halfghaninne) [inline lib/conditionals.md]: Can you say more about what "setting the parameters" means? "Parameter" is a word that gets used itself in programming so I want to make sure I fully understand your answer.

Comment (michellecali) [inline lib/conditionals.md]: How about, "it sets the rules for which parts of the code will be executed."

Does that work? I want to say "decision rules", but fear we will run into the same issue. :)

Comment (michellecali) [inline lib/conditionals.md]: I resubmitted, adding a little bit more. Thanks!

**Comment by halfghaninne** [comment]
That looks great! Thank you!

**Pull request closed without merging**